opentasks/src/main/java/org/dmfs/tasks/SettingsListFragment.java

```diff
@@ -524,7 +524,7 @@ public void doneSaveListState()
 
 
     /**
-     * This class is used to display a list of accounts. The list can be modified by the {@link #addAccount(Account)} and {@link #clear()} method. The dialog is
+     * This class is used to display a list of accounts. The list can be modified by the  and  method. The dialog is
```

Comment:
The second sentence doesn't make much sense anymore now. Maybe just remove it altogether.